Running the Android example app fails with a flipper error

Trying to run the `example` on Android TV throws the following exception:

```
E/SoLoader: couldn't find DSO to load: libflipper.so caused by: dlopen failed: cannot locate symbol "_Unwind_Resume" referenced by "/data/app/com.reactnativetheoplayer-q9EeEDXLB83p_FfRf7E25A==/lib/x86/libflipper.so"... result: 0
E/AndroidRuntime: FATAL EXCEPTION: FlipperConnectionThread
    Process: com.reactnativetheoplayer, PID: 3402
    java.lang.NoClassDefFoundError: <clinit> failed for class com.facebook.flipper.android.EventBase; see exception in other thread
        at com.facebook.flipper.android.FlipperThread.run(FlipperThread.java:25)
E/AndroidRuntime: FATAL EXCEPTION: FlipperEventBaseThread
    Process: com.reactnativetheoplayer, PID: 3402
    java.lang.UnsatisfiedLinkError: couldn't find DSO to load: libflipper.so caused by: dlopen failed: cannot locate symbol "_Unwind_Resume" referenced by "/data/app/com.reactnativetheoplayer-q9EeEDXLB83p_FfRf7E25A==/lib/x86/libflipper.so"... result: 0
        at com.facebook.soloader.SoLoader.doLoadLibraryBySoName(SoLoader.java:918)
        at com.facebook.soloader.SoLoader.loadLibraryBySoNameImpl(SoLoader.java:740)
        at com.facebook.soloader.SoLoader.loadLibraryBySoName(SoLoader.java:654)
        at com.facebook.soloader.SoLoader.loadLibrary(SoLoader.java:634)
        at com.facebook.soloader.SoLoader.loadLibrary(SoLoader.java:582)
        at com.facebook.flipper.android.EventBase.<clinit>(EventBase.java:19)
        at com.facebook.flipper.android.FlipperThread.run(FlipperThread.java:25)
     Caused by: java.lang.UnsatisfiedLinkError: dlopen failed: cannot locate symbol "_Unwind_Resume" referenced by "/data/app/com.reactnativetheoplayer-q9EeEDXLB83p_FfRf7E25A==/lib/x86/libflipper.so"...
        at java.lang.Runtime.load0(Runtime.java:938)
        at java.lang.System.load(System.java:1631)
        at com.facebook.soloader.SoLoader$1.load(SoLoader.java:405)
        at com.facebook.soloader.DirectorySoSource.loadLibraryFrom(DirectorySoSource.java:77)
        at com.facebook.soloader.DirectorySoSource.loadLibrary(DirectorySoSource.java:50)
        at com.facebook.soloader.ApplicationSoSource.loadLibrary(ApplicationSoSource.java:89)
        at com.facebook.soloader.SoLoader.doLoadLibraryBySoName(SoLoader.java:860)
        at com.facebook.soloader.SoLoader.loadLibraryBySoNameImpl(SoLoader.java:740) 
        at com.facebook.soloader.SoLoader.loadLibraryBySoName(SoLoader.java:654) 
        at com.facebook.soloader.SoLoader.loadLibrary(SoLoader.java:634) 
        at com.facebook.soloader.SoLoader.loadLibrary(SoLoader.java:582) 
        at com.facebook.flipper.android.EventBase.<clinit>(EventBase.java:19) 
        at com.facebook.flipper.android.FlipperThread.run(FlipperThread.java:25) 
```

It appears that Flipper is misconfigured in this project.

--- a/example/android/app/src/debug/AndroidManifest.xml
+++ b/example/android/app/src/debug/AndroidManifest.xml
@@ -1,13 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" xmlns:tools="http://schemas.android.com/tools">
 
-    <uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW"/>
+    <uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW" />
 
-    <application
-        android:usesCleartextTraffic="true"
-        tools:targetApi="28"
-        tools:ignore="GoogleAppIndexingWarning">
-        <activity android:name="com.facebook.react.devsupport.DevSettingsActivity" />
-    </application>
+    <application android:usesCleartextTraffic="true" tools:targetApi="28" tools:ignore="GoogleAppIndexingWarning" />
 </manifest>

--- a/example/android/gradle.properties
+++ b/example/android/gradle.properties
@@ -25,7 +25,7 @@ android.useAndroidX=true
 android.enableJetifier=true
 
 # Version of flipper SDK to use with React Native
-FLIPPER_VERSION=0.75.1
+FLIPPER_VERSION=0.105.0
 
 # Version of the THEOplayer SDK, if not specified, the latest available version is set.
 # THEOplayer_sdk=3.4.0

